frontend/src/resources/theme.js

```diff
@@ -1,37 +1,74 @@
 /**
  * The theme provides cross component UI like Colors and Fonts.
+ * @author Irem Toroslu
+ * @author Martin Wagner 
  */
 
 const color = {
     brightBlue: '#3498db',
+    white:'white',
     darkGrayishBlue: '#8b8d94',
+    whitish: '#b5b4b9',
+    white:'#000000',
+    darkGray: '#262625',
     darkRed: '#a90000',
     grayishBlue: '#A4A6B3',
     grayishBlue2: '#9fa2b4',
     grayishBlue3: '#bdc3c7',
+    limeGreen: '#00b300',
+    blueGray:'#33b3a6',//687f8c016064018788019799
     lightBlue: '#3751FF',
     lightGrayishBlue: '#F7F8FC', // background color
     lightGrayishBlue2: '#DFE0EB',
     paleBlue: '#DDE2FF',
+    darkBlue:'#008caf',
     paleBlueTransparent: 'rgba(221, 226, 255, 0.08)',
-    veryDarkGrayishBlue: '#373a47'
+    veryDarkGrayishBlue: '#373a47',
+    lightgreen: '#66a103',
+    lightblueGray:'#2e9db0',
+    lightGreenGray:'#9dccca',
+    GreenGray:'#88b598',
+    darkGreenGray:'#94b0a5',//009092,94b0a5 5ba68880a7af
+    darkskyblue:'#01a1c1',//1a1c1
+    Copper:'#BF6016',//B87333
+    Alminium:'#6093AC',//'#A9ACB6',
+    SiliconSteel:'#0c6c84',//'#8FB1CC',016064,8EA0B3, 016A86,006994,04657e,0c6c84
+    TransformerOil: '#FACA0F',//'#D4AF37 cca42b',
+    StainlessSteel:'#caccce', //
+    Pressboard:'#C38888',//BC8379,
```

Comment:
If these commented out values do mean something, they probably shouldn't be commented out, otherwise they should be gone 😜 

```suggestion
    darkGreenGray:'#94b0a5',
    darkskyblue:'#01a1c1',
    Copper:'#BF6016',
    Alminium:'#6093AC',
    SiliconSteel:'#0c6c84',
    TransformerOil: '#FACA0F',
    StainlessSteel:'#caccce',
    Pressboard:'#C38888',
```